test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -3372,6 +3372,48 @@ def forward(self, x, k):
         k = torch.tensor(3)
         self.run_test(MyModuleDynamic(), [x, k])
 
+    @disableScriptTest()
+    def test_auto_grad(self):
+        class MyClip(torch.autograd.Function):
+            @staticmethod
+            def forward(ctx, input, min):
+                ctx.save_for_backward(input)
+                return input.clamp(min=min)
+
+        class MyRelu(torch.autograd.Function):
+            @staticmethod
+            def forward(ctx, input):
+                ctx.save_for_backward(input)
+                return input.clamp(min=0)
+
+        from torch.onnx.symbolic_helper import _unimplemented
+
+        def symbolic_pythonop(g: torch._C.Graph, n: torch._C.Node, *args, **kwargs):
```

Comment:
here and in the onnx.rst file, please change from `symbolic_pythonop` to `symoblic_python_op`.
In general the translation between CamelCase and snake_case should preserve the separation between words.